### DISCUSSION TITLE
[QUESTION] Bootstrap 5

### DISCUSSION THREAD

**Discussion by lenamtl** [ColorlibHQ/AdminLTE#3909] [Q&A]
Hi,

I have a project that still use AdminLTE  v2.3.1

I don't know when AdminLTE  v4 is planned....
But I will need to update the template soon.

I'm wondering if anyone have manage to update AdminLTE  v2.3.1 to use Bootstrap 5 ?

Thanks

**Comment by REJack** [comment]
1. Don't use the Bug Report for Questions, for this we have GitHub's discussions or our discord.
2. You can update AdminLTE 2.x to use Bootstrap 5 but this will cost much work to get it working without bugs

**Reply by lenamtl** [reply]
Sorry for that, I didnt realized that there is a the discussion menu fir this...
You can remove the bug label too.
